org.serioussam.SeriousSamClassic-VK.yaml

```diff
@@ -0,0 +1,118 @@
+app-id: org.serioussam.SeriousSamClassic-VK
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: serioussam.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+
+modules:
+
+  - name: serioussam
+    buildsystem: cmake-ninja
+    no-make-install: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX:PATH="/usr"
```

Comment:
this is 100%% wrong. 

```suggestion
```

Reply:
This option can simply be removed. Does not affect the build.